Frontend/src/pages/registration_administration/components/RegistrationAdministrationList.jsx

```diff
@@ -44,6 +45,16 @@ const selectedReducer = (state, action) => {
   return newState
 }
 
+const sortReducer = createSortReducer([
```

Comment:
People like to sort by comment, for example for a multi-location FM competition where you need to put your location in the comment.
And what about DOB? I'm not sure how useful that is, but it's possible in the existing registration.